samples/matter/lock/src/app_task.cpp

```diff
@@ -152,50 +100,9 @@ CHIP_ERROR AppTask::Init()
 	/* Initialize lock manager */
 	BoltLockMgr().Init(LockStateChanged);
 
-#ifdef CONFIG_CHIP_OTA_REQUESTOR
-	/* OTA image confirmation must be done before the factory data init. */
-	OtaConfirmNewImage();
-#endif
+	ReturnErrorOnFailure(Nordic::Matter::StartChipServer());
```

Comment:
Could `StartChipServer` call `WaitForReadiness` internally? It's weird to me that user has to remember about calling some synchronization mechanisms of our module.